meson.build

```diff
@@ -74,7 +70,7 @@ add_project_arguments(
   '-DG_PASTE_ICON_NAME="edit-paste"',
   '-DGDK_DISABLE_DEPRECATED',
   '-DGETTEXT_PACKAGE="GPaste"',
-  '-DGTK_API=3.0',
+  '-DGTK_API=3',
```

Comment:
I'd prefer if we leave it as 3.0

Reply:
That makes the build fail:

../src/libgpaste/gpaste-gtk-compat.h:14:5: error: floating point literal in preprocessor expression
#if GTK_API > 3

Reply:
Right, my bad